src/Tests/Microsoft.NET.Sdk.Razor.Tests/RazorSdkTest.cs

```diff
@@ -0,0 +1,39 @@
+using Microsoft.NET.TestFramework.Commands;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+using System.Text;
+using System.Threading;
+using System.Xml.Linq;
+using Microsoft.NET.TestFramework;
+using Xunit.Abstractions;
+
+namespace Microsoft.NET.Sdk.Razor.Tests
+{
+    public abstract class RazorSdkTest : SdkTest
+    {
+        public readonly string DefaultTfm = "net6.0";
```

Comment:
You could stash the tfm of the test project as an assembly attribute and use that here. Here's what that would look like: https://github.com/dotnet/aspnetcore/blob/master/src/Razor/Microsoft.NET.Sdk.Razor/integrationtests/Microsoft.NET.Sdk.Razor.IntegrationTests.csproj#L61-L66

https://github.com/dotnet/aspnetcore/blob/44c0e6651e38079cada6e14ac8978cdb9fd59cc1/src/Shared/MSBuild.Testing/BuildVariables.cs#L14

Of course this is assuming the test tfm gets updated every release.

Reply:
This is a great idea! It means we can be mostly hands-free when it comes to TFM updates in the repo.

I've updated the code to use this approach.